libraries/wallet/include/golos/wallet/remote_node_api.hpp

```diff
@@ -213,12 +254,17 @@ FC_API( golos::wallet::remote_node_api,
         (get_account_reputations)
         (get_reblogged_by)
         (get_blog_authors)
+)
+
+/**
+ * Declaration of remote API formatter to follow plugin on remote node
+ */
+FC_API( golos::wallet::remote_market_history,
```

Comment:
I guess it might be a good idea to make a separate file for every remote API here.